sdk/formrecognizer/Azure.AI.FormRecognizer/src/ReadingOrder.cs

```diff
@@ -1,14 +1,30 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-namespace Azure.AI.FormRecognizer.Models
-{
-    // Setting the "internal" access modifier while we don't expose the reading order
-    // feature as part of our design.
+using Azure.Core;
 
-    internal enum ReadingOrder
+namespace Azure.AI.FormRecognizer
+{
+    /// <summary>
+    /// Specifies the order in which recognized text lines are returned in calls to
+    /// recognize content. This can be set in the <see cref="RecognizeContentOptions"/>.
```

Comment:
The plan is to have ReadingOrder in every endpoint, so maybe we should remove the `recognize content` part